operations/CSM_product_management/Configure_Non-Compute_Nodes_with_CFS.md

```diff
@@ -45,7 +45,15 @@ with CFS.
 ### Option 2: Provide Custom SSH Keys
 
 Administrators may elect to replace the CSM-provided keys with their own custom
-keys.
+keys.  The `replace_ssh_keys.sh` script can be used to replace the keys from
```

Comment:
Nit: still using double-spacing here between the sentences, but should be single spaced

Reply:
Apologies, I've fixed all instances of this now.